tools/TestCoverageAnalysis/AnalyzeTestCoverage.ps1

```diff
@@ -0,0 +1,169 @@
+$repoRootDirectory = $PSScriptRoot | Split-Path | Split-Path
```

Comment:
- Please add [license notice](https://github.com/Azure/azure-powershell/blob/main/tools/CheckAssemblies.ps1) at top of the file.
- Also it would be nice to add doc about how to use this script. [Here's an example](https://github.com/Azure/azure-powershell/blob/main/tools/StaticAnalysis/ExampleAnalyzer/Measure-MarkdownOrScript.ps1)
- We should consider calling this script from pipeline, so that we can validate its correctness.

Reply:
License and comments added
The script will be combined with the GitHub bot. It likely will be updated again.